bin/dialyzer_check.sh

```diff
@@ -1,9 +1,13 @@
+#!/bin/bash
+
 # currently dialyzer is timing out after 30 mins only on elixir 1.7 / otp 19
 if elixir --version | grep -q 'Elixir 1.7'; then
   echo "skipping dialyzer for Elixir 1.7"
   exit 0
 fi
 
+mkdir -p ./priv/plts
+
 mkdir -p ./tmp/src
 mkdir ./tmp/build
 
```

Comment:
This may require some sort of caching step in our action, due to the slowness of dialyzer

Reply:
Agreed. GHA has some caching actions would should be applicable. Looks like a 5 minute runtime at the moment (all of it in generating the plt).

Reply:
There has been discussion prior whether or not we should even do a dialyzer check.  Perhaps now is the time to ditch it? @NobbZ @Cohen-Carlisle @devonestes

Reply:
PLT caching makes this insignificant,  nvm!